.ci/packer_cache.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+_sdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# shellcheck disable=SC1091
+source /usr/local/bin/bash_standard_lib.sh
+
+readonly GO_VERSION=$(cat ${_sdir}/../.go-version)
```

Comment:
The script runs in the context of the workspace, therefore if .go-version is in the top level folder, you could simplify the _sdir 

```
 readonly GO_VERSION=$(cat .go-version)
```